.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI ARC E2E Linux VM Test
+
+on:
+  workflow_dispatch:
+  
+jobs:
+  setup-steps:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Go is pre-installed on the runners, no?

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
Yes, was not aware, thank you!